lib/templates/bootstrap/select.ios.js

```diff
@@ -55,7 +55,7 @@ class CollapsiblePickerIOS extends React.Component {
             }, animationConfig)).start();
             this.setState({isCollapsed: !this.state.isCollapsed});
           }}>
-          <Text style={pickerValue}>
+          <Text numberOfLines={1} style={pickerValue}>
```

Comment:
Can you update your PR to bring support for `numberOfLines` through a `locals.config` option?